content/release/10.0.1.md

```diff
@@ -0,0 +1,112 @@
+---
+Title: FRRouting Release 10.0.1
+Release: 10.0.1
+GitHub: https://github.com/FRRouting/frr/releases/tag/frr-10.0.1
+Date: 2024-06-06
+---
+
+FRRouting Release 10.0.1 Available for Download
+=============================================
+
+We are pleased to announce FRR release 10.0.1.
+
+FRR 10.0.1 brings a long list of enhancements and fixes with 938 commits from 54 developers. Thanks to all contributors.
+
+Debian Packages - https://deb.frrouting.org
+
+RPM Packages - https://rpm.frrouting.org
+
+Snaps - https://snapcraft.io/frr
+
+Docker - [quay.io/frrouting/frr:10.0.1](https://quay.io/repository/frrouting/frr/manifest/sha256:572c60d3d678887bbd97281e7995a976f7eceeb7f40bf43a13cbd7905d52af41)
```

Comment:
The link needs to be updated after the images are built.

Reply:
https://quay.io/repository/frrouting/frr/manifest/sha256:8ab9fe2ded0be732634bbd9a38e69c72ad450c7c9c41289b05a5ae7e26fdbe8b